test/uk/gov/hmrc/filetransmission/services/RetryQueueBackedTransmissionServiceSpec.scala

```diff
@@ -17,6 +17,8 @@
 package uk.gov.hmrc.filetransmission.services
 
 import java.net.URL
+import java.sql.Timestamp
```

Comment:
Is this used?  If not it should be removed.

Reply:
Removed, thanks!